app/models/event.rb

```diff
@@ -211,6 +211,15 @@ def speaker_names
     result.to_a.to_sentence
   end
 
+  # Returns emails of all the speaker belongs to a particular event
+  def speaker_emails
+    result = []
+    speakers.each do |speaker|
+      result << speaker.email
+    end
+    result.to_sentence
```

Comment:
This can be done simply like this:
`speakers.map(&:email).join(', ')`

Reply:
I think i can do same in the speaker_names